### ISSUE TITLE
Client should warn about insufficient stake when registering a baker

### ISSUE THREAD

**Issue by concordium-cl** [Concordium/concordium-client#80]
**Description**
Client should warn when registering as baker with insufficient stake. Right now, the transaction fails eventually without warning. In contrast, when updating stake, it talks a lot about the correct amount etc.

![stake too low](https://user-images.githubusercontent.com/69570978/134354076-7d553297-0f26-4043-93e3-bf96a7667740.JPG)

**Steps to Reproduce**
- register as baker with insufficient stake

**Expected Result**
Client informs before submitting to chain.

**Actual Result**
Client does not warn before submitting to chain.

**Versions**
- CLI 1.1.1-0
- Windows

**Comment by abizjak** [comment]
Good catch. I removed the "Windows" label since this is a cross-platform deficiency.
